Fixes broken link for minka- logreg

shrankhla20 opened the pull request:
#25024 issue

<!--
Thanks for contributing a pull request! Please ensure you have taken a look at
the contribution guidelines: https://github.com/scikit-learn/scikit-learn/blob/main/CONTRIBUTING.md
-->

#### Reference Issues/PRs
<!--
Example: Fixes #1234. See also #3456.
Please use keywords (e.g., Fixes) to create link to the issues or pull requests
you resolved, so that they will automatically be closed when your pull request
is merged. See https://github.com/blog/1506-closing-issues-via-pull-requests
-->


#### What does this implement/fix? Explain your changes.
removed slash after the pdf link and it started working

#### Any other comments?


<!--
Please be aware that we are a loose team of volunteers so patience is
necessary; assistance handling other issues is very welcome. We value
all user contributions, no matter how minor they are. If we are slow to
review, either the pull request needs some benchmarking, tinkering,
convincing, etc. or more likely the reviewers are simply busy. In either
case, we ask for your understanding during the review process.
For more information, see our FAQ on this topic:
http://scikit-learn.org/dev/faq.html#why-is-my-pull-request-not-getting-any-attention.

Thanks for contributing!
-->

shrankhla20 committed:
Fixed broken link for minka- logreg

#25024 issue

lesteve reviewed (approved):
Indeed, thanks!

~For completeness sake, this link was flagged as broken by `make linkcheck` but the link on the documentation is fine.~

**Edit**: actually the link was broken in the doc https://scikit-learn.org/dev/modules/linear_model.html#logistic-regression. I was looking at the wrong page.

shrankhla20 committed:
Fixes broken link for yu-shi msc

Fixed broken link for multiclass spectral clustering yu-shi reference 
#25024 issue

lesteve commented:
@shrankhla20 you just pushed a commit fixing another link, can you open a separate PR for it? 

One PR per link makes it easier to track the advancement of the issue :pray:

lesteve committed:
Revert "Fixes broken link for yu-shi msc"

This reverts commit 61be65ad83f8073edb70e10c19ea308af297dd5a.

lesteve commented:
@shrankhla20 I reverted your last commit and merged your PR, thanks a lot!

Feel free to open a separate PR for the other link you were working on.

Here is your diff in case it helps:
```diff
commit 61be65ad83f8073edb70e10c19ea308af297dd5a (HEAD -> shrankhla)
Author: Shrankhla Srivastava <shrankhla.srivastava@gmail.com>
Date:   Fri Nov 25 16:28:43 2022 +0530

    Fixes broken link for yu-shi msc
    
    Fixed broken link for multiclass spectral clustering yu-shi reference
    #25024 issue

diff --git a/doc/modules/clustering.rst b/doc/modules/clustering.rst
index 40ca15aaa..5ca3a6f46 100644
--- a/doc/modules/clustering.rst
+++ b/doc/modules/clustering.rst
@@ -532,7 +532,7 @@ below.
 .. topic:: References:
 
  * `"Multiclass spectral clustering"
-   <https://www1.icsi.berkeley.edu/~stellayu/publication/doc/2003kwayICCV.pdf>`_
+   <https://people.eecs.berkeley.edu/~jordan/courses/281B-spring04/readings/yu-shi.pdf>`_
    Stella X. Yu, Jianbo Shi, 2003
 
  * :doi:`"Simple, direct, and efficient multi-way spectral clustering"<10.1093/imaiai/iay008>`
```

shrankhla20 commented:
@lesteve Thank you so much. 

P.s. This marks my first contribution to open source !

lesteve commented:
Great stuff!
